host-configs/TOTAL/pangea2-gcc@8.3.0.cmake

```diff
@@ -18,9 +18,9 @@ set(CMAKE_C_COMPILER "/data_local/sw/gcc/RHEL7/8.3.0/bin/gcc" CACHE PATH "")
 set(CMAKE_CXX_COMPILER "/data_local/sw/gcc/RHEL7/8.3.0/bin/g++" CACHE PATH "")
 set(CMAKE_Fortran_COMPILER "/data_local/sw/gcc/RHEL7/8.3.0/bin/gfortran" CACHE PATH "")
 
-set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG -march=native -mtune=native" CACHE STRING "")
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG -march=native -mtune=native" CACHE STRING "")
-set(CMAKE_Fortran_FLAGS_RELEASE "-O3 -DNDEBUG -march=native -mtune=native" CACHE STRING "")
+set(CMAKE_C_FLAGS_RELEASE "-O3 -DNDEBUG -march=x86-64 -mtune=generic" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3 -DNDEBUG -march=x86-64 -mtune=generic" CACHE STRING "")
+set(CMAKE_Fortran_FLAGS_RELEASE "-O3 -DNDEBUG -march=x86-64 -mtune=generic" CACHE STRING "")
```

Comment:
I am not certain that these are the most correct values to use. For instance, we could be specific about the chip for `march`. @TotoGaz Do you know that the chip is? I don't recall.

Also, I think `mtune` is redundant.

https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html

Reply:
The fact is that the login nodes of Pangea not necessarily have the same CPU as the computation nodes.
We could probably explicitly put the correct target architecture, and let developers trick this file if needed while developing.

Reply:
You can build on a compute node? I think that is what I typically did.

Reply:
You can but it's not obvious. You need to document the proper `bsub/brun` command line to be able to connect, and besides you may have to request some dedicated permissions. Plus if you want to run/debug a little local case, you may want to compile run locally as well.

I do not think you did compile on compute node but on login nodes (`pandev*` are somehow Pangea login nodes). But some other graphical login nodes (`pangraph*`) do not share the same CPU.